source/MapPanel.h

```diff
@@ -185,17 +187,21 @@ class MapPanel : public Panel {
 	// This is the coloring mode currently used in the cache.
 	int cachedCommodity = -10;
 
+	BatchDrawList batchDraw;
```

Comment:
Why is this a class field when it is only ever used in `MapPanel::DrawSystems` and it is not used to store any information across mmultiple calls to that function (it is cleared after every time it is used before the function returns)?

Reply:
It seems to me that it can be a local variable in the method where it is used.